bootstrap.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -exu
+
+die () {
+    echo >&2 "$@"
+    exit 1
+}
+
+[ "$#" -eq 1 ] || die "1 argument required, $# provided. Pleas add the project name"
+
+project_name=$1
+
+# install ruby gems
+bundle install
+
+#install node packages
+yarn
+
+# copy secrets from the example adding a new application secret
+application_secret=$(bundle exec rake secret)
+sed "s/SECRET_KEY_BASE:/SECRET_KEY_BASE: ${application_secret}/g" config/application.yml.example > config/application.yml
+
+# copy database configuration and change the project name with the given one
+sed "s/sample_project/${project_name}/g" config/database.yml.example > config/database.yml
+
+# setup database
+bundle exec rails db:create db:migrate db:seed
```

Comment:
```suggestion
bundle exec rails db:setup
```
Does exactly the same

Reply:
Yeah but instead of running migrations, it loads the schema. My concern is that if migrations break no one will realize